lib/gsheets.js

```diff
@@ -176,8 +176,8 @@ async function parseInput(rawInput, startDate = null, numDays = 5, slotsInDay) {
  * Read and parse agent preferences and availability from Support Scheduler History sheet.
  * @param  {object} auth           OAuth 2.0 access token
  * @param  {string} nextMondayDate Schedule start date in format YYYY-MM-DD
- * @param  {number} numDays        Number of consecutive days to schedule
- * @return {object}                Parsed input object for scheduler (more options will be added to this object by download-and-configure-input.js)
+ * @param  {object} support
```

Comment:
Yes, typescript validates the param names in the jsdoc actually match up to param names in the function declaration :P